src/components/CallView/shared/VideoVue.vue

```diff
@@ -22,8 +22,12 @@
 	<div v-show="!placeholderForPromoted || sharedData.promoted"
 		:id="(placeholderForPromoted ? 'placeholder-' : '') + 'container_' + peerId + '_video_incoming'"
 		ref="videoContainer"
-		class="videoContainer"
-		:class="containerClass"
+		class="video-container"
+		:class="[containerClass, {
+			speaking: isSpeaking,
```

Comment:
```suggestion
			speaking: isSpeaking && !isBig,
```
Agree with excluding the stripe, but Big is already promoted, so feels weird to have an outline around id